Xamarin.Essentials/TextToSpeech/TextToSpeech.macos.cs

```diff
@@ -51,7 +51,7 @@ internal static async Task PlatformSpeakAsync(string text, SpeechOptions options
 
             void TryCancel()
             {
-                ss.StopSpeaking(NSSpeechBoundary.hWord);
+                ss.StopSpeaking(NSSpeechBoundary.Word);
```

Comment:
Did they fix this bug?

Reply:
I think this is literally a big in the API: https://docs.microsoft.com/en-us/dotnet/api/appkit.nsspeechboundary?view=xamarin-mac-sdk-14